src/content/en/2020/seo.md

```diff
@@ -83,493 +83,603 @@ Response Code | All Devices | Mobile | Desktop
 
-####  Images
+#### Images
 
-The usage of images, particularly using `<img>` elements, within a page often suggests a focus on visual presentation of content. Although search engine capabilities regarding computer vision have continued to improve, we have no indication that this technology is being used in the ranking of pages. `alt` attributes remain the primary way to explain an image in lieu of a search engine’s inability to "see" it. `alt` attributes also support accessibility and clarify the elements on the page for users that are visually impaired.
+The usage of images, particularly using `img` tags, within a page often suggests a focus on visual presentation of content. Although search engine capabilities regarding computer vision have continued to improve, we have no indication that this technology is being used in the ranking of pages. `alt` attributes remain the primary way to explain an image in lieu of a search engine's ability to "see" it. `alt` attributes also support accessibility and clarify the elements on the page for users that are visually impaired.
 
-The median desktop page features 21 image tags and the median mobile page features 19 image tags. The web continues to trend toward image-heaviness with the growth of bandwidth and the ubiquity of smartphones. However, this comes at a cost of performance.
+{# TODO(authors): Why are the growth of bandwidth and ubiquity of smartphones contributors to image heaviness? #}
+The median desktop page includes 21 `img` tags and the median mobile page has 19 `img` tags. The web continues to trend toward image-heaviness with the growth of bandwidth and the ubiquity of smartphones. However, this comes at a cost of performance.
 
 {{ figure_markup(
   image="seo-img-elements-per-page.png",
   alt="img elements per page",
-  caption="`<img>` elements per page",
+  caption="Distribution of the number of `<img>` elements per page.",
   description="Bar chart showing the number of `<img>` elements per page per percentile (10, 25, 50, 75, and 90). The median desktop page features 21 `<img>` elements and the median mobile page features 19 `<<img>` tags..",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=923860709&format=interactive",
   sheets_gid="1317352222",
   sql_file="pages_markup_by_device_and_percentile.sql"
-  )
-}}
+) }}
 
-The median web page is missing 2.99% `alt` attributes on desktop and 2.44% of `alt` attributes on mobile.
+{# TODO(authors): Add your interpretation of these results. What do you hope readers get from it? Also consider linking to the Accessibility chapter here if that ties in. #}
```

Comment:
@aleyda

Reply:
Thanks for the heads up @rviscomi - @ipullrank could you please take a look at this regarding the "alt" attributes usage interpretation?

@rviscomi could you please let us know where is the Accessibility chapter to link?

Reply:
You could link to it like this:

```
For more information on the importance of `alt` attributes, see the [Accessibility](../accessibility) chapter.
```

Reply:
FYI @rviscomi just happened to spot you linked it wrong (two dots instead of one) so corrected your comment.

Reply:
Thanks, good catch!

Reply:
Added the Accessibility chapter link but leaving the TODO unresolved for more interpretation